schemas/common/extensible.schema.json

```diff
@@ -288,5 +302,5 @@
       "$ref": "#/definitions/@context"
     }
   ],
-  "meta:status": "stable"
-}
\ No newline at end of file
+  "meta:status": "stabilizing"
```

Comment:
Don't move from stable -> stabalizing

Reply:
It was always "stablizing" I guess. I will change it.